### PR TITLE
Remove Visit.previous_visit

### PR DESCRIPTION
**Pull request by jamienoss**
Resolves [Notion ticket](https://www.notion.so/Remove-Visit-previous_visit-48c4b017cc1b41c2be7c525abcee56b0?pvs=4)

### CODE REVIEW

**Review by glemson — COMMENTED** [review]
really only one question

File: biospecdb/apps/uploader/admin.py [hunk 1]
```diff
@@ -473,26 +473,18 @@ def get_extra(self, request, obj=None, **kwargs):
...
+    fields = ("patient", "patient_age")
 
     @admin.display
     def patient_id(self, obj):
         return obj.patient.patient_id
 
     @admin.display
     def visit_count(self, obj):
-        return obj.visit_number
+        return Visit.objects.filter(patient=obj.patient).count()
```
[the remark this answers is not in the document]
Comment (jamienoss) [reply biospecdb/apps/uploader/admin.py:487]: Correct. ``Visit`` isn't been removed, if it were then we wouldn't be able to have multiple visits per patient. All that's being removed in this PR is ``Visit.previous_visit`` which gave a mechanism for directly linking visits and thus explicitly chaining their order.

**Review by vkluzner — APPROVED** [review]
Looks good, the only question is: what's the reason to remove this field? My impression was it provided valuable insight about the full patient info (when you can see previous visits).

**Comment by jamienoss** [comment]
@vkluzner Previous visits will still exist as we're not nuking the ``Visit`` table but just ``Visit.previous_visit``. So, yes, while we won't be tracking explicitly which visit came prior to another (via this linking), the info is still there.

We discussed this with the PI and decided to nuke this field since in most cases it's probably not going to get buy-in from the hospitals contributing the data.

**Pull request merged**